packages/examples/fortune/recording-oracle/tests/routes/index.test.ts

```diff
@@ -0,0 +1,18 @@
+import server from '../../src/server';
+import { describe, test, expect } from 'vitest';
+
```

Comment:
I am missing here some test with data that does not pass validation

Reply:
I will start working on tests later and take a look